# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xorg/driver-enable.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

I have a PowerPC board with Cirrus chip here, the driver compiles - no idea
it it works out-of-the-box ...

  - Rene Rebe <rene@exactcode.de>

--- ./config/cf/xorg.cf.orig	2005-01-17 18:13:00.730993000 +0100
+++ ./config/cf/xorg.cf	2005-01-17 18:11:52.121423240 +0100
@@ -619,7 +619,7 @@
 /* Pure PCI drivers should go first */
 # ifndef XF86CardDrivers
 #  define XF86CardDrivers	ati mga glint nv s3 s3virge sis savage\
-				trident chips tdfx \
+				cirrus trident chips tdfx \
 				DevelDrivers vga \
 				XF86OSCardDrivers XF86ExtraCardDrivers
 # endif
